conduit/pipeline/pipeline.go

```diff
@@ -574,8 +696,7 @@ func MakePipeline(ctx context.Context, cfg *data.Config, logger *log.Logger) (Pi
 		return nil, fmt.Errorf("MakePipeline(): could not build importer '%s': %w", importerName, err)
 	}
 
-	importer := importerBuilder.New()
-	pipeline.importer = &importer
+	pipeline.importer = importerBuilder.New()
```

Comment:
This change and the ones below in this file aren't needed, but seem ok.

Reply:
Well, partially their needed, but we don't really need to remove the extra variable assignment as I'm doing here.

Reply:
Not sure why this variable is named "Builder". This is the "Constructor" and "New" is the single function defined in the constructor interface. The function above should be `importers.ImporterConstructorByName`.

What you have here is correct, previously it had to be assigned separately so that we could get the pointer.

Reply:
Thanks. Up to you, I don't feel very strongly. Was just calling out a possible reason for the discrepancy because you mentioned it in your top comment.

Reply:
Went ahead and merged. We should discuss how to communicate this for our next release.